lib/widgets/task_item_widget.dart

```diff
@@ -41,17 +45,43 @@ class TaskItemWidget extends StatelessWidget {
             child: Column(
               crossAxisAlignment: CrossAxisAlignment.start,
               children: [
-                Text(
+                text.isNotEmpty ? Text(
                   text,
                   style: MainStyles.kBlackColor1W500(20.0),
-                ),
-                reminderTime != '' ? Padding(
-                  padding: const EdgeInsets.only(top: 10.0),
-                  child: Text(
-                    Strings.reminder + reminderTime,
-                    style: MainStyles.kBlackColor1W500(16.0),
-                  ),
                 ) : const SizedBox(),
+                pathsToPictures.isNotEmpty
+                    ? Padding(
+                        padding: const EdgeInsets.symmetric(vertical: 6.0),
+                        child: SizedBox(
+                          height: 100.0,
+                          child: Row(
+                            children: pathsToPictures.map((element) {
+                              return Expanded(
+                                child: Padding(
+                                  padding: const EdgeInsets.symmetric(horizontal: 2.0),
+                                  child: ClipRRect(
+                                    borderRadius: BorderRadius.circular(6.0),
+                                    child: Image.file(
+                                      File(element),
+                                      fit: BoxFit.fitHeight,
+                                    ),
+                                  ),
+                                ),
+                              );
+                            }).toList(),
```

Comment:
Try to create specific widgets for cases like this, so that you will avoid huge widgets